src/app/application.cpp

```diff
@@ -126,9 +126,8 @@ namespace
 #endif
 }
 
-Application::Application(const QString &id, int &argc, char **argv)
+Application::Application(int &argc, char **argv)
     : BaseApplication(argc, argv)
-    , m_instanceManager(new ApplicationInstanceManager {id, this})
```

Comment:
A bit off topic to this PR.
We should assign a default `nullptr` to this variable in header.